### DISCUSSION TITLE
How can I have page number in Table of content (TOC) ?

### DISCUSSION THREAD

**Discussion by happytodev** [honkit/honkit#316] [Q&A]
Hello,

My TOC is like below : 

![SCR-20221210-wea](https://user-images.githubusercontent.com/425998/206877547-f75c8f71-e4a5-4b28-9f2d-056034d79275.png)

Instead of 1.1, 1.2 I wish to have the real page number.

How can I do this please ?

Thank you.

**Comment by azu** [comment]
HonKit use [ebook-convert](https://manual.calibre-ebook.com/generated/en/ebook-convert.html) to create PDF.
If [ebook-convert](https://manual.calibre-ebook.com/generated/en/ebook-convert.html) has the option, It will be resolved.
(I do not know details)

**Reply by happytodev** [reply]
Hello, thank you for your answer.
Sadly, I didn't seen anything for that in the TOC section of ebook-convert.
But I'm pretty sure I had already seen a TOC with page number, so there is probably a solution for that.
